docs/static/settings/configuration-wildcard-pipeline-id.asciidoc

```diff
@@ -0,0 +1,19 @@
+[role="xpack"]
+[[wildcard-in-pipeline-id]]
+==== Wildcard support in pipeline ID
+++++
+<titleabbrev>Wildcard support in pipeline ID</titleabbrev>
+++++
+Pipeline ID must begin with a letter or underscore and contain only letters, underscores, dashes, and numbers.
+You can use `*` in `xpack.management.pipeline.id` to match any number of letter, underscore, dash, and number.
+
+[source,shell]
+-----
+xpack.management.pipeline.id: ["*logs", "*apache*", "tomcat_log"]
+-----
+
+In this example, `"*logs"` matches all IDs end in `logs`. `"*apache *"` matches any IDs with `apache` in the name.
+
+Wildcard is available starting with Logstash 7.11 alongside Elasticsearch 7.10. Logstash can pick up new pipeline without restart Logstash if the new pipeline ID matches the wildcard pattern.
```

Comment:
We try to avoid listing version numbers in documentation.  Documentation is available (using the version dropdown) for all versions, so this info is generally not needed.  For example, this docs PR would be merged into `master` and `7.x`, but no earlier version so we don't have to say it starts at Logstash 7.11 because it won't be in docs for Logstash 7.10 and earlier.  

Sometimes we make exceptions based on the situation. Let's discuss if you'd like. :-)

Reply:
totally agree. Should I mention Elasticsearch version? Do we expect users to have the same version in general, so we don't mention?